src/jsifier.js

```diff
@@ -279,7 +286,7 @@ function ${name}(${args}) {
       }
 
       if (symbolsOnly) {
-        if (!isJsOnlySymbol(symbol) && LibraryManager.library.hasOwnProperty(symbol)) {
```

Comment:
Is this change related?

Reply:
Yup, this is because I removed the `INCLUDE_FULL_LIBRARY` hack in emcc.py.

Now we include JS symbols in the dependency info file too.    This way INCLUDE_FULL_LIBRARY can specify all possible native deps.. even native deps on JS symbols.